packages/base/glob.js

```diff
@@ -6,6 +6,7 @@ import { Readable } from 'readable-stream'
 import tracer from './lib/tracer.js'
 
 function glob({ pattern, ...options }) {
+  const { logger } = this
```

Comment:
is it worth covering also the cases in which `logger` (or even `this`) is undefined?

Reply:
No. That should never happen